File: 20170418/northwindService/src/main/java/com/test/invoice/Invoice.java
package com.test.invoice;

import java.math.BigDecimal;
import java.util.Date;

public class Invoice {

	public Long id;
	public Long orderId;
	public Date invoiceDate;
	public Date dueDate;
	public BigDecimal tax;
	public BigDecimal shipping;
	public BigDecimal amountDue;
	
	@Override
	public String toString() {
		return "Invoice [id=" + id + ", orderId=" + orderId + ", invoiceDate=" + invoiceDate + ", dueDate=" + dueDate
				+ ", tax=" + tax + ", shipping=" + shipping + ", amountDue=" + amountDue + "]";
	}
	
}

File: 20170418/northwindService/src/app/invoices/invoice.ts
export class Invoice {

    public id : number;
    public orderId : number;
    public invoiceDate : number;
    public dueDate : number;
    public tax : number;
    public shipping : number;
    public amountDue : number;
    
    constructor(values: Object = {}) {
        Object.assign(this, values);
    }    
}
